### ISSUE TITLE
Add metadata to .root output file

### ISSUE THREAD

**Issue by lepmik** [dekamps/miind#13]
As of now, the root file does not tell you which population gives which rate etc. The names from the xml files would be nice to pass on.

**Comment by dekamps** [comment]
I agree the populations should be named in the root file, but this is not really metadata. New feature in development.
